### PR TITLE
Tidy a few minor errors and add some important notes

### PR DESCRIPTION
**Pull request by NightJar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,11 +128,15 @@ Now that we've installed SilverStripe, let's finely tune our development environ
...
 
 It can also include other application settings. You might have API keys or email addresses in there that you want to specify as globally accessible by all projects. 
 
-The key attribute of the `.env` file, however, is that it does not have to ship with the project. It can live outside the web root, outside of source control. When you deploy this project from your local environment to somewhere else, that remote environment might have its own configuration, so having the file outside the project means you don't have to worry about overriding settings.
+The key attribute of the `.env` file, however, is that it should not ship with the project. It can live outside the web root, outside of source control. When you deploy this project from your local environment to somewhere else, that remote environment should have its own configuration, so having the file outside the project (or at least ignored by your source control system) means you don't have to worry about accidentally overriding settings.
+
+<div class="alert alert-danger">
```
Comment (robbieaverill) [inline README.md:137]: Typo: environments

So SS4 doesn't have any equivalent of \_ss_environment.php for production? That seems like a huge oversight to me.

File: README.md [hunk 2]
```diff
@@ -156,6 +160,11 @@ You can place an `.env` file in, say project B, and it will override the parent
     * project-a/
     * project-b/ [ .env ]*
     * project-c/
+	
+
+<div class="alert alert-info">
+	The `.env` file is searched for in the project root, and then it's parent directory only. Only the first file found is loaded.
```
Comment (robbieaverill) [inline README.md]: its*

File: README.md [hunk 3]
```diff
@@ -196,17 +205,19 @@ Some of the fields have been populated for you, such as the database username an
...
+### Going further
+
+Let's now take this a step further. There are some more things we want to throw into our `.env` file. We can use `SS_DATABASE_CHOOSE_NAME` to tell SilverStripe to intelligently determine a database name so that you don't have to. It will look at the filesystem, see where the project is installed, and choose a database name based on that. By default this takes the form of e.g. `SS_project-a`, keeping with the examples above.
 
-Also, you can specify the default admin username and password. For local development, you're probably not too concerned about security. So having something easy to remember, like *root/root*, is just fine. 
+Also, you can specify the default admin username and password. For local development you're probably not too concerned about security, So having something easy to remember such as *root/root*, is just fine.
```
Comment (robbieaverill) [inline README.md]: so*

File: README.md (hunk 1 above)
Comment (NightJar) [reply README.md:137]: Fixed. And yeah, `.env` is it. But in production one should not be using it, instead opting to set environment variables via e.g. `SetEnv` in Apache, or the fast cgi set params directive for nginx, etc. Or just having them in the environment before starting the process, if the server only runs a single site.

**Review by robbieaverill — APPROVED** [review]

**Pull request merged**